ui.frontend/src/view/FormFieldBase.js

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+import Utils from "../utils";
+import FormField from './FormField'
+
+export default class FormFieldBase extends FormField {
```

Comment:
its still not clear why there are two base class: Field and BaseField. is there any kind of guidance on the methods to be included in either of the classes?